balancer-js/src/lib/utils/errors.ts

```diff
@@ -12,7 +12,7 @@ const balancerErrorCodes: Record<string, string> = {
   '100': 'OUT_OF_BOUNDS',
   '101': 'UNSORTED_ARRAY',
   '102': 'UNSORTED_TOKENS',
-  '103': 'INPUT_LENGTH_MISMATCH',
```

Comment:
Hey @sergioyuhjtman - I'm sorry I said it would be ok to replace the error message, but looking at this list, I believe this error should not be replaced as this is a Balancer Error - [see here](https://dev.balancer.fi/references/error-codes). Could you please revert this change?